update_kb_redis.py

```diff
@@ -569,9 +568,9 @@ def process_keyboard(keyboard, usb_list, kb_list, kb_entries):
 
     # Setup platform specific keys
     mcu = rules_mk.get('MCU')
-    if mcu in ARM_PROCESSORS:
+    if mcu.startswith(CHIBIOS_PROCESSORS):
```

Comment:
I don't like this, it allows invalid processors to be specified:

```
>>> 'STM32F103doesnotexist'.startswith(CHIBIOS_PROCESSORS)
True
```

Reply:
How else would be be able to specify `MCU = STM32F303xC` without adding every size variant to both here and mcu_selection.mk?

Reply:
I think we should not try to use one key for two purposes that require differing formats.